rancid-tomatillos/cypress/e2e/movie_view.cy.js

```diff
@@ -1,21 +1,49 @@
-describe('empty spec', () => {
-  it('passes', () => {
-    cy.visit('https://example.cypress.io')
+describe('MovieView Test Suite', () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/508439")
+  })
+
+  it("should be able to visit the app and display the Navbar and its contents", () => {
+    cy
+      .get("header")
+      .contains("Rancid Tomatillos")
+      .should("have.length", 1)
+      .get(".btn").contains("HOME")
   })
-})
 
-//1)should display the User sees new page 
-//displaying the selected movie's information 
-//(movie image, statistics, genres) 
-//2) Nav Bar displays Web page name
-//2) no longer can see any other movies
+  it("Should be able to see all the details about the movie they clicked on", () => {
+    cy
+      .get(".movieViewMain").should("have.attr", "src", "https://image.tmdb.org/t/p/original//xFxk4vnirOtUxpOEWgA1MCRfy6J.jpg")
+      .get(".title").contains("Onward")
+      .get(".tagline").contains("Their quest begineth.")
+      .get(".releaseDate").contains("2020-02-29")
+      .get(".budget").contains("200000000")
```

Comment:
I wonder if there is a way to dry this up? I know the data comes through as an array, and I wonder if there is any way to test for that data type differently. We can research this a bit in the next iteration.